ocp4/product.yml

```diff
@@ -3,6 +3,10 @@ full_name: Red Hat OpenShift Container Platform 4
 type: platform
 
 benchmark_root: "../applications"
+additional_content_directories:
+  - "../linux_os/guide/system/software/integrity/endpoint_security_software/mcafee_security_software"
+  - "../linux_os/guide/system/network/network-ipv6/disabling_ipv6"
+  - "../linux_os/guide/system/selinux"
```

Comment:
This doesn't sound right... why are these being included?

Reply:
That's a good catch. So we have some "shared" OVALs that don't belong to any rules. One would expect them to be included in the benchmark if they are needed by rule checks, but that's not the case - they are included unconditionally. Those incriminated definitions are the `` and `` ones.
The quick fix of these cases would be to set "affected platform" of those snippets to Linux OSs only, as they don't make sense for OCP, OSP and other application-level products.

In the longer run, we should do something about our messed-up system of platforms vs prodtypes vs products.
Finally, those shared checks should be included only on-demand, and there should be a distinction between an applicability of the check and whether the check is actually needed by some other check or by an existing rule.